scripts/run_tutorials.py

```diff
@@ -26,19 +26,20 @@
     "vae_mnist.ipynb",  # requires setting paths to local data
     "bope.ipynb",  # flaky, keeps failing the workflows
     "preference_bo.ipynb",  # failing. Fix planned
+}
+IGNORE_SMOKE_TEST_ONLY = {  # only used in smoke tests
+    "thompson_sampling.ipynb",  # very slow without KeOps + GPU
+    "composite_mtbo.ipynb",  # TODO: very slow, figure out if we can make it faster
+    "Multi_objective_multi_fidelity_BO.ipynb",  # TODO: very slow, speed up
+}
+IGNORE_STANDARD_ONLY = {
```

Comment:
I find this naming very confusing. `IGNORE_SMOKE_TEST_ONLY` means to run it in smoke test only -- maybe we should just call it that `RUN_SMOKE_TEST_ONLY`?

For `IGNORE_STANDARD_ONLY`, the comment says "# Causing the tutorials to crash when run without smoke test. Likely OOM.". Does that mean we also run these only in smoke test, in which case they should be in the previous list?

Reply:
OK, this should be clearer now. There actually were never any tutorials we ran only in standard mode. I was confused about this and probably so was whoever put those tutorials on the ignore list in the first place, since they actually run fine in smoke test mode. I moved those tutorials to the list now called "RUN_IF_SMOKE_TEST_IGNORE_IF_STANDARD"